src/components/operations/operation-details/ko/runtime/operation-details.html

```diff
@@ -175,12 +175,33 @@ <h4>Request body</h4>
                     <!-- /ko -->
 
                     <!-- ko if: representation.examples -->
+                    <!-- ko if: representation.examples.length > 1 -->
+                        <div class="row flex flex-row example-form-control">
+                            <div class="col-12">
+                                <label data-bind="attr : { for: 'select' + representation.contentType + '_' + $index() }, text: 'Examples'"></label>
+                            </div>
+                            <div class="col-4">
+                                <div class="form-group">
+                                    <select class="form-control" data-bind="options: representation.examples, value: $component.selectedRepresentatnionsValue()[representation.contentType], event: {change: $component.permissionChanged} ,optionsValue: 'title', optionsText: 'title', attr : {title: 'Represantation examples selector', id: 'select-' + representation.contentType + '_' + $index() }"></select>
+                                </div>
+                            </div>
+                        </div>
+                        <!-- ko foreach: { data: representation.examples, as: 'example' } -->
```

Comment:
what is the purpose of multiple "ko ..." existing in the file?

Reply:
as comments

Reply:
This is Knockout framework syntax to create repeated html elements. for example create lists of elements which are dynamically fetched from server

Reply:
this is knockout js code, which executes as part of the page. when you need some logic there to hide something to loop the array